src/StatusQ/Components/StatusSeedPhraseInput.qml

```diff
@@ -0,0 +1,154 @@
+            //        //        }
+            //            }
+        }
+    }
+
+    Item {
+        width: seedSuggestionsList.width
+        height: seedSuggestionsList.height
+        x: !!grid.currentItem ? mapFromItem(grid.currentItem, 0, grid.currentItem.height).x + 16 : 0
+        y: !!grid.currentItem ? mapFromItem(grid.currentItem, 0, grid.currentItem.height).y + 16 : 0
+        visible: (seedSuggestionsList.count > 0)
+        Rectangle {
+            id: statusPopupMenuBackgroundContent
+            anchors.fill: parent
+            color: Theme.palette.statusPopupMenu.backgroundColor
+            radius: 8
+            layer.enabled: true
+            layer.effect: DropShadow {
+                anchors.fill: parent
+                source: statusPopupMenuBackgroundContent
+                horizontalOffset: 0
+                verticalOffset: 4
+                radius: 12
+                samples: 25
+                spread: 0.2
+                color: Theme.palette.dropShadow
+            }
+        }
+        ListView {
+            id: seedSuggestionsList
+//          width: ((txtDelegate.suggWordCntWidth + 28) > parent.width) ? parent.width
+//                 : (txtDelegate.suggWordCntWidth + 28)
+            width: 80
+            height: ((count*34) + 16)
+            clip: true
+            delegate: Item {
+                id: txtDelegate
+                width: parent.width
+                height: 34
+                Rectangle {
+                    anchors.fill: parent
+                    color: sensor.containsMouse? Theme.palette.statusPopupMenu.hoverBackgroundColor : "transparent"
+                }
+                StatusBaseText {
+                    id: suggWord
+                    anchors.fill: parent
+                    anchors.leftMargin: 14
+                    anchors.rightMargin: 14
+                    verticalAlignment: Text.AlignVCenter
+                    text: seedWord
+                    color: Theme.palette.directColor1
+                    font.pixelSize: 13
+                    elide: Text.ElideRight
+                }
+                MouseArea {
+                    id: sensor
+                    anchors.fill: parent
+                    cursorShape: Qt.PointingHandCursor
+                    hoverEnabled: true
+                    onClicked: {
+//                    grid.currentItem.input.edit.clear();
+//                    grid.currentItem.text = seedWord;
+                    }
+                }
+            }
+        }
+    }
+
+    StatusButton {
+        anchors.horizontalCenter: parent.horizontalCenter
+        anchors.top: grid.bottom
+        anchors.topMargin: 66
+        text: ((grid.count > 12) && !grid.secondPage) ? qsTr("Next") : qsTr("Import")
+        onClicked: {
+            if ((grid.count > 12) && !grid.secondPage) {
+               grid.positionViewAtEnd();
+            } else {
+                //import seed word to text input
+            }
+        }
+    }
+}
```

Comment:
Hey @alexandraB99 ,
sorry don't wanna get ahead of myself here, but just noticing: I don't think we want the seed phrase input the entire page, but really just the input + its suggestions thingy.

The page will be built in desktop. This also allows for using StatusSeedPhraseInput in different ways.